### DISCUSSION TITLE
overwrite current session

### DISCUSSION THREAD

**Discussion by iomari** [tmux-python/tmuxp#722] [Q&A] [answered]
greetings,
I'm using a zsh plugin called zsh4humans which automatically starts with a tmux session.
My problem is that when I run tmuxp, it complains that there is already a tmux session running. Is there an option that tmuxp can overwrite the current session?

thanks
iomari

**Comment by tony** [comment, marked as answer]
@iomari Can you try `tmux list-sessions`. Is the session name automatically started the same as the automatically launched session?

How about you add `-a` to `tmuxp load`. e.g. `tmuxp load myfile.yaml -a` Is it any better?

P.S. I converted this to a Q&A discussion, hopefully that's okay!
